src/nodes/chunk_append/chunk_append.c

```diff
@@ -38,6 +38,27 @@ has_joins(FromExpr *jointree)
 	return list_length(jointree->fromlist) != 1 || !IsA(linitial(jointree->fromlist), RangeTblRef);
 }
 
+static void
+process_current_group(PlannerInfo *root, RelOptInfo *rel, List *group, List *pathkeys,
```

Comment:
Could you use a more fitting name for the function (e.g., `create_group_path`)?

Reply:
renamed to `create_group_subpath`